src/screens/EPub/models/data_reducer.js

```diff
@@ -303,11 +303,18 @@ export default {
         chapters[chapterIdx].title = value;
         return { ...state, epub: { ...state.epub, ...nextStateOfChapters([...chapters]) } };
     },
-    splitChaptersByScreenshots(state) {
+    splitChaptersByScreenshots(state, {payload: {wc}}) {
         console.log(`Splitting chapters by screenshots`);
         const new_items = [];
+        const N = wc;
         (state.items).forEach(function(elem) {
-            if((elem.text).length < 100 && new_items.length!==0 ) { 
+            let words = 1;
+            for (let i = 0; i < (elem.text).length; i+=1) {
+                if((elem.text).charAt(i)===' ') {
```

Comment:
A better way to count word count of a string would be to do str.split(' ').length